### DISCUSSION TITLE
What iPhone model?

### DISCUSSION THREAD

**Discussion by francisferras** [fullpwn/infinity#3] [Q&A] [answered]
Hi there,

I’m a newbie all around, but would like to try your software in my old iPhone 5.

is it possible? Any guide to assist me/others available?

thanks

**Comment by missaustraliana** [comment, marked as answer]
Hi,
Thank you for your interest in my software. 
**infinity only supports iPhone 7 , 7 Pus and iPod Touch 7th Gen.** 
If you cannot meet those requirements, this software will not work. 

Thanks,
fullpwn
